security/src/main/java/io/micronaut/security/handlers/ForbiddenRejectionUriProvider.java

```diff
@@ -29,8 +31,8 @@
 public interface ForbiddenRejectionUriProvider {
 
     /**
-     *
+     * @param request {@link HttpRequest} being processed
      * @return A uri to redirect to when an authenticated user tries to access a resource for which he does not have the required authorization level.
      */
-    Optional<String> getForbiddenRedirectUri();
+    Optional<String> getForbiddenRedirectUri(HttpRequest<?> request);
```

Comment:
This is a breaking change? Or is this new in 1.1

Reply:
> `ForbiddenRejectionUriProvider.java` is new in 1.1